applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_explicit_solver.py

```diff
@@ -0,0 +1,115 @@
+
+# Import base class file
+from KratosMultiphysics.ConvectionDiffusionApplication import (
+    convection_diffusion_base_solver,
+)
+
+
+def CreateSolver(model, custom_settings):
+    return ConvectionDiffusionExplicitSolver(model, custom_settings)
+
+
+class ConvectionDiffusionExplicitSolver(
+    convection_diffusion_base_solver.ConvectionDiffusionBaseSolver
+):
+    """
+    The explicit class for convection-diffusion solvers.
+    See convection_diffusion_base_solver.py for more information.
+    """
+
+    def __init__(self, model, custom_settings):
+        # Construct the base solver and validate the remaining settings in the base class
+        super(ConvectionDiffusionExplicitSolver, self).__init__(model, custom_settings)
+
+        # Overwrite the base solver minimum buffer size
+        self.min_buffer_size = 2
+
+        element_name = self.settings["element_replace_settings"][
+            "element_name"
+        ].GetString()
+        if self.settings["use_orthogonal_subscales"].GetBool() is True:
+            if element_name in (
+                "QSConvectionDiffusionExplicit",
+                "DConvectionDiffusionExplicit",
+            ):
+                self.main_model_part.ProcessInfo.SetValue(
+                    KratosMultiphysics.OSS_SWITCH, 1
+                )
+            else:
+                err_msg = self.__class__.__name__
+                err_msg += (
+                    "The selected element",
+                    element_name,
+                    "does not support OSS projection. Select QSConvectionDiffusionExplicit or DConvectionDiffusionExplicit instead.",
+                )
+                raise Exception(err_msg)
+        else:
+            if element_name in (
+                "QSConvectionDiffusionExplicit",
+                "DConvectionDiffusionExplicit",
+            ):
+                self.main_model_part.ProcessInfo.SetValue(
+                    KratosMultiphysics.OSS_SWITCH, 0
+                )
+            else:
+                err_msg = self.__class__.__name__
+                err_msg += (
+                    "The selected element",
+                    element_name,
+                    "does not support OSS projection. Select QSConvectionDiffusionExplicit or DConvectionDiffusionExplicit instead.",
+                )
+                raise Exception(err_msg)
+
+        KratosMultiphysics.Logger.PrintInfo(
+            self.__class__.__name__, "Construction finished"
+        )
+
+    @classmethod
+    def GetDefaultParameters(cls):
+        default_settings = KratosMultiphysics.Parameters(
+        """
+        {
+            "use_orthogonal_subscales" : false,
+            "explicit_parameters" : {
+                "dynamic_tau": 1.0
+            }
+        }
+        """
+        )
+
+        default_settings.AddMissingParameters(super().GetDefaultParameters())
+        return default_settings
+
+    #### Private functions ####
+
+    def _create_builder_and_solver(self):
```

Comment:
```suggestion
    def _CreateBuilderAndSolver(self):
```
Style guide. Apply same suggestion to the other methods.

Reply:
The convection diffusion base solver uses `_create_builder_and_solver`... So I cannot apply these changes.

Reply:
We should change it at some point.